deltachat-ios/Controller/ContactDetailViewController.swift

```diff
@@ -122,9 +125,15 @@ class ContactDetailViewController: UITableViewController {
         super.viewWillAppear(animated)
         updateHeader() // maybe contact name has been edited
         updateCellValues()
+        setupObservers()
```

Comment:
in general, to not miss an update, the observer needs to be added beforehand - otherwise, an event may arrive between ui update and overserver adding and ui will show the wrong values.

might be a _very_ minor thing here only, however, these patterns tend to be copy+pasted, cmp. https://github.com/deltachat/deltachat-ios/pull/1338, therefore i suggest to to that right ;)